CHANGELOG.md

```diff
@@ -153,11 +181,58 @@ options to operations that previously had none. Users should migrate to these
+* fix(storage): json["projectTeam"] might be present but null ([#8446](https://github.com/googleapis/google-cloud-cpp/pull/8446))
+* feat(GCS+gRPC): implement `SetBucketIamPolicy()` ([#8431](https://github.com/googleapis/google-cloud-cpp/pull/8431))
+* doc(storage): improve `{Bucket,Object}Metadata` docs ([#8434](https://github.com/googleapis/google-cloud-cpp/pull/8434))
+* feat(GCS+gRPC): implement `TestBucketIamPermissions()` ([#8427](https://github.com/googleapis/google-cloud-cpp/pull/8427))
+* feat(GCS+gRPC): implement `LockBucketRetentionPolicy()` ([#8428](https://github.com/googleapis/google-cloud-cpp/pull/8428))
+* feat(GCS+gRPC): implement `GetNativeBucketIamPolicy()` ([#8420](https://github.com/googleapis/google-cloud-cpp/pull/8420))
+* feat(GCS+gRPC): implement `UpdateBucket()` ([#8416](https://github.com/googleapis/google-cloud-cpp/pull/8416))
+* feat(GCS+grpc): implement `ListBuckets()` ([#8415](https://github.com/googleapis/google-cloud-cpp/pull/8415))
+* feat(GCS+gRPC): implement `PatchBucket()` ([#8394](https://github.com/googleapis/google-cloud-cpp/pull/8394))
+* fix(storage): ignored fields in lifecycle patches ([#8389](https://github.com/googleapis/google-cloud-cpp/pull/8389))
+* fix(GCS+gRPC): ignored fields in BucketMetadata parsing ([#8390](https://github.com/googleapis/google-cloud-cpp/pull/8390))
+* feat(GCS+gRPC): implement `DeleteBucket()` ([#8375](https://github.com/googleapis/google-cloud-cpp/pull/8375))
+* feat(GCS+gRPC): implement `CreateBucket()` ([#8372](https://github.com/googleapis/google-cloud-cpp/pull/8372))
+* doc(storage): show how to configure endpoints ([#8354](https://github.com/googleapis/google-cloud-cpp/pull/8354))
+* feat(GCS+gRPC): implement `UpdateObject()` ([#8278](https://github.com/googleapis/google-cloud-cpp/pull/8278))
+* feat(GCS+gRPC): implement `ComposeObject()` ([#8209](https://github.com/googleapis/google-cloud-cpp/pull/8209))
+* doc(storage): UserIp option is deprecated ([#8468](https://github.com/googleapis/google-cloud-cpp/pull/8468))
+
+### [Common Libraries](https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/README.md)
+
+* feat(generator): all bidir streams are experimental ([#8471](https://github.com/googleapis/google-cloud-cpp/pull/8471))
+* doc: update GA library list  ([#8467](https://github.com/googleapis/google-cloud-cpp/pull/8467))
+* feat: better support for PSC and VPC-SC ([#8453](https://github.com/googleapis/google-cloud-cpp/pull/8453))
+* fix: wait until AutomaticallyCreatedBackgroundThreads start ([#8452](https://github.com/googleapis/google-cloud-cpp/pull/8452))
+* fix(oauth2): compute engine credential query ([#8421](https://github.com/googleapis/google-cloud-cpp/pull/8421))
+* fix: restore Options over deletion of StreamRange<T>::reader_ ([#8403](https://github.com/googleapis/google-cloud-cpp/pull/8403))
+* feat(common): ddd converting constructors for future ([#8329](https://github.com/googleapis/google-cloud-cpp/pull/8329))
```

Comment:
I think `ddd` is a typo, just remove please.

Reply:
Done.